### ISSUE TITLE
Feature request: Docker Support for ARM to be able to use raspberrypi+docker for hosting

### ISSUE THREAD

**Issue by thinkpace** [nicksherron/bashhub-server#21]
I would like to host bashhub-server via Docker on a Raspberry Pi, which is based on ARM. It would be really cool if you add docker support for ARM.

**Comment by nicksherron** [comment]
Good idea! I just pushed an image to docker hub, would you mind testing it on your Raspberry Pi and letting me know if it is working alright? 

`docker pull nicksherron/bashhub-server:issue-21`

@thinkpace

**Comment by thinkpace** [comment]
Short test with Raspberry Pi 4 Model B Rev 1.4 -> works totally great! Thanks!

Test cases:

- [x] Container startup
- [x] Register new user and link device
- [x] Link new device to existing user
- [x] Add new command on different devices
- [x] Lookup for commands on different devices
- [x] No errors in docker log

Test system:
Raspbian GNU/Linux 10 (buster)
Linux amalthea 5.10.17-v7l+ #1403 SMP Mon Feb 22 11:33:35 GMT 2021 armv7l GNU/Linux
Bashhub client 2.1.3 (python 3.7.3)

**Comment by francescor** [comment]
Hi, we badly need the arm version, too (the arm64, aka aarch64) to use this in arm based Ec2 instances in AWS.

I see you already release arm version, it is just docker hub without it (even the `nicksherron/bashhub-server:issue-21`)

```
$ docker manifest inspect nicksherron/bashhub-server | grep -i arch 
$ docker manifest inspect nicksherron/bashhub-server:issue-21 | grep -i arch
```

We are available for testing if you need 


thanks!
